installer/roles/image_build/files/supervisor.conf

```diff
@@ -46,8 +46,20 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
+[program:awx-rsyslogd]
+command = rsyslogd -n -i /var/run/rsyslog/rsyslog.pid -f /var/lib/awx/rsyslog/rsyslog.conf
+autostart = true
+autorestart = true
+stopwaitsecs = 1
+stopsignal=KILL
```

Comment:
So this will send a `SIGKILL` then wait 1 second, then send a `SIGKILL` again. Do we want to be this forceful ?

Reply:
🤷 it's what we do for our other services?  I suppose we could give it a bit more time?  It's the duration we give Python process, though.

Reply:
Yea, given the nature of what rsyslog does, maybe we should extend this a bit to something like 5 seconds.

cc @rooftopcellist ✅ let's track this adjustment as a task post-merge